subjects/blockchain/node-dashboard/README.md

```diff
@@ -1,43 +1,42 @@
-# A network dashboard and benchmarking tool
+## A network dashboard and benchmarking tool
 
-Blockchain nodes participate in a common peer-to-peer network retrieving information, executing and validating transactions and blocks. However, software clients differ in their functionalities and resource consumption. The purpose of this project is to create a tool, `benchy` that can launch, monitor and benchmark Ethereum networks. 
+Blockchain nodes participate in a common peer-to-peer network retrieving information, executing and validating transactions and blocks. However, software clients differ in their functionalities and resource consumption. The purpose of this project is to create a tool, `benchy` that can launch, monitor and benchmark Ethereum networks.
 
-## launch-network
+### launch-network
 
 At first, we will focus on deploying a private Ethereum network while monitoring the nodes. The nodes can be launched directly or within containers.
 
-* The network should consist of five nodes named Alice, Bob, Cassandra, Driss, and Elena. The first three are validating nodes. Each node has a corresponding Ethereum address.
-* Two different clients must be used, for instance, Geth and Nethermind
-* The consensus mechanism must be clique. 
+- The network should consist of five nodes named Alice, Bob, Cassandra, Driss, and Elena. The first three are validating nodes. Each node has a corresponding Ethereum address.
+- Two different clients must be used, for instance, Geth and Nethermind
+- The consensus mechanism must be clique.
 
 The network should be launched with the command `benchy launch-network`. The nodes should be launched in the background and the command should return immediately.
 
-## Infos
+### Infos
 
 Using the `infos` command, Benchy should display information on each node, in the terminal or in a webpage. The information must include:
 
-* Their latest block
-* Their list of connected peers
-* The number of transactions in their mempool
-* Their current CPU and memory consumption (Hint: the outputs of commands such as `docker stats` or `ps` can be used)
-* Their corresponding Ethereum address and its balance.
+- Their latest block
+- Their list of connected peers
+- The number of transactions in their mempool
+- Their current CPU and memory consumption (Hint: the outputs of commands such as `docker stats` or `ps` can be used)
+- Their corresponding Ethereum address and its balance.
 
+### Scenario
 
-## Scenario
+To analyze our network, we need to simulate some transactions. Benchy should propose the following scenarios available with the command `benchy scenario X`. Each scenario must provide feedback on its execution, including updated balances after transfers. 0. Initialise the network by letting it run for a few minutes. Validating nodes must have Eth available as reward or part of the initial configuration.
```

Comment:
here i think the "0. Initialise the network by letting it run for a few minutes. Validating nodes must have Eth available as reward or part of the initial configuration."  needs to be in the next line like the others, and maybe substitute the number with normal  -

Reply:
I implemented everything you suggested! :)